app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/ContractDetailActivity.kt

```diff
@@ -32,6 +35,7 @@ class ContractDetailActivity : BaseActivity(R.layout.contract_detail_activity) {
             e { "Programmer error: ID not provided to ${this.javaClass.name}" }
             return
         }
+        model.data.observe(this) { it.bindTo(binding.card) }
         model.loadContract(id)
```

Comment:
The arrow is still visible and the card is clickable here

Reply:
Good catch. 😬